install.sh

```diff
@@ -191,7 +191,14 @@ echo_out 'Set up the local package repo...'
 # Download the chromebrew repository.
 curl_wrapper -L --progress-bar https://github.com/"${OWNER}"/"${REPO}"/tarball/"${BRANCH}" | tar -xz --strip-components=1 -C "${CREW_LIB_PATH}"
 
-BOOTSTRAP_PACKAGES='glibc_standalone libxcrypt upx patchelf lz4 zlib xzutils zstd zlib_ng crew_mvdir ruby git ca_certificates libyaml openssl gmp'
+BOOTSTRAP_PACKAGES=
+ZSTD_STATUS=
+if ! zstd --help &>/dev/null; then
+  ZSTD_STATUS="broken"
```

Comment:
Surprised rubocop didn't complain about the double quotes.

Reply:
This is shellcode, not ruby.

Reply:
> Surprised rubocop didn't complain about the double quotes.

Didn't realize this was a bash shell script.